pkg/utils/files.go

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func IsYaml(file string) bool {
+	ext := strings.ToLower(filepath.Ext(file))
+	return ext == ".yaml" || ext == ".yml"
+}
+
+func IsJson(file string) bool {
+	ext := strings.ToLower(filepath.Ext(file))
+	return ext == ".json"
+}
+
+func IsYamlOrJson(file string) bool {
+	return IsYaml(file) || IsJson(file)
+}
+
+func FindFiles(args ...string) ([]string, error) {
```

Comment:
@alexzielenski how do you feel about the behavior always being recursive? Should we need a `-r` to make it recursive? I don't know if that makes sense ... I'm personally fine with it that way.

Reply:
we talked about that in another thread. I think it should be always recursive.